CHANGELOG.md

```diff
@@ -15,7 +15,9 @@
 
 #### Bug Fixes
 
-* None.
+* Fix the regex for expiring-todos  
+  [namolnad](https://github.com/namolnad)
+  [#3597](https://github.com/realm/SwiftLint/issues/3597)
```

Comment:
can you change the link here to `/pull/3597` (instead of _issue_)? Alternatively, it's fine to remove this line completely since most of times we haven't been linking to pull requests, only issues

Reply:
Updated, thanks!